src/rev_ai/apiclient.py

```diff
@@ -184,14 +195,23 @@ def submit_job_local_file(
             Sections of transcript needed to be transcribed.
         :param speaker_names: Only available with "human" transcriber.
             Human readable names of speakers in the file.
+        :param notification_config: CustomerUrlData object containing the callback url to
+            invoke on job completion as a webhook and optional authentication headers to use when
+            calling the callback url
         :returns: raw response data
-        :raises: HTTPError
+        :raises: HTTPError, ValueError
```

Comment:
If you let API validate then you can remove ValueError

Reply:
Do you still need the value error?

Reply:
I think so, because we are still validating that they submit a non-null filename in this case. We were raising the ValueError but it wasn't in the doc. We don't need to remove that specific validation for this ticket since it doesn't use source_config or media_urll